package.json

```diff
@@ -8,6 +8,6 @@
   },
   "scripts": {
     "start": "gulp clean && gulp compile && gulp theme",
-    "postinstall": "rm -rf pattern-lab && composer create-project -n pattern-lab/edition-twig-standard pattern-lab && rm -rf pattern-lab/source && ln -s ../components pattern-lab/source"
+    "postinstall": "rm -rf pattern-lab && composer create-project -n pattern-lab/edition-twig-standard pattern-lab && rm -rf pattern-lab/source && ln -s ../components pattern-lab/source && git clone https://github.com/drupal-pattern-lab/bem-twig-extension.git && cp bem-twig-extension/bem.function.php components/_twig-components/functions && rm -rf bem-twig-extension/"
```

Comment:
This is a great idea - I created an issue over on emulsify-gulp and assigned you but feel free to move wherever https://github.com/fourkitchens/emulsify-gulp/issues/34